v2/azure-arm.yaml

```
@@ -2431,6 +2436,19 @@ objectModelConfiguration:
       WebTestGeolocation:
         Id:
           $armReference: false
+    2021-05-01-preview:
+      DiagnosticSetting:
+        $export: true
+        $supportedFrom: v2.10.0
```

As mentioned above, we just missed the v2.10 release.

```suggestion
        $supportedFrom: v2.11.0
```

---

Updated to `v2.11.0`